src/main/java/core/basesyntax/service/impl/FileWriterImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileWriter;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.IOException;
+
+public class FileWriterImpl implements FileWriter {
+
+    private final File file;
+
+    public FileWriterImpl(String fileName) {
+        this.file = new File(fileName);
+    }
+
+    @Override
+    public void write(String report, String fileName) {
```

Comment:
The `write` method takes a `fileName` parameter, but it is not used because the `file` is already initialized in the constructor with a specific file name. Consider either removing the `fileName` parameter from the `write` method or using it to dynamically set the file to which the report is written.